views/layout.php

```diff
@@ -52,23 +114,126 @@
         </div>
 
         <div class="container" style="background-color:#FAFC2F">
-            <button type="button" onclick="document.getElementById('id01').style.display='none'" class="cancel-btn">Cancel</button>
+            <button type="button" onclick="document.getElementById('id01').style.display='none'" class="cancel-btn">
+                Cancel
+            </button>
             <span class="psw">Forgot <a href="#">password?</a></span>
 
         </div>
     </form>
 </div>
 
+<!--Registration Modal Placement-->
```

Comment:
I think it is better to have the registration.php instead of modal.

Reply:
[ ] Remove the registration as modal. We did not have an agreement to make this as a modal